packages/tests/src.ts/test-wallet.spec.ts

```diff
@@ -4,16 +4,59 @@ import assert from "assert";
 
 import { ethers } from "ethers";
 import { loadTests, TestCase } from "@ethersproject/testcases";
+import { BytesLike } from "@ethersproject/bytes";
 import * as utils from './utils';
 import { arrayify, getAddressFromAccount, Logger } from "ethers/lib/utils";
 import {
+    AccountCreateTransaction, AccountId,
+    Client,
     ContractCreateTransaction,
     ContractExecuteTransaction,
     FileAppendTransaction,
-    FileCreateTransaction, PublicKey, Transaction
+    FileCreateTransaction, Hbar, PrivateKey, PublicKey, Transaction, TransactionId,
+    Key as HederaKey
 } from "@hashgraph/sdk";
 import { readFileSync } from "fs";
 import * as hethers from "ethers";
+import { Wallet } from "./../../wallet";
+import { Key } from "@hashgraph/proto";
+import { BaseProvider } from "./../../providers";
```

Comment:
nit: you can access `Wallet` and `BaseProvider` via `ethers.Waller` and `ethers.getDefaultProvider('network_name')`

Reply:
Resolved